docs/ad-hoc-mainframe-deployment.md

```diff
@@ -0,0 +1,129 @@
+## Ad-Hoc deployment of API Mediation Layer to Mainframe system
+
+## Technology
+
+Using [zowe-api-dev](https://github.com/zowe/sample-spring-boot-api-service/blob/master/zowe-rest-api-sample-spring/docs/devtool.md
+) tool from Zowe SDK.
+
+Underlying technology is Zowe CLI.
+
+**WARNING: TECHNOLOGY AND IMPLEMENTATION IS STILL IN EXPERIMENTAL STAGE**
+
+### Prerequisities
+ - Install Zowe CLI - https://docs.zowe.org/stable/user-guide/cli-installcli.html#installing-zowe-cli
+ - Install zowe-api-dev
+   ```
+   npm -g install @zowedev/zowe-api-dev
+   ```
+ - Create Zowe CLI profiles for Z/OSMF and USS
+   ```
+   zowe profiles create zosmf-profile <profile_id> --host <hostname> --port <port> --user <userid> --pass "<password>" --reject-unauthorized false
+   zowe profiles create ssh-profile <profile_id> --host <hostname> --user <userid> --password "<password>"
+   ```
+### Procedure
+
+Procedure is separated into 3 distinct phases
+
+*All commands are run from repository root folder*
+
+#### Phase 1 - Initial run
+
+objective of this phase is to setup zowe-api-dev and perform first deployment
+
+ - Run 'zowe-api-dev init --account=<mf-account-id>'. 'user-zowe-api.json' will be generated in repository root folder.
+
+  - Create `user-zowe-api.json` file in repository root folder. Fill in specific information. 
```

Comment:
```suggestion
  - Update the `user-zowe-api.json` file in the repository root folder. Fill in specific information. 
```

I would use _update_ since the `user-zowe-api.json` is already created by `zowe-api-dev` init.

Reply:
good point, i need to adjust, but the other way around

Reply:
We would want to init with custom parameters. Since it's not available and you need to edit the file anyway, it's without init at the moment